pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/util/DdpInstanceGroupTestUtil.java

```diff
@@ -116,8 +116,10 @@ private static void deleteInstance(int instanceId) {
 
     public static int getDdpInstanceId(String instanceName) {
```

Comment:
Although this method name isn't being changed directly in this PR, I'll mention that a getter is not expected to create something--it should just return a thing.  I think it's more clear to have the "create if not exists" embedded in the method name.  Something like "createDDPInstanceIfNotFound".